src/tv2-common/cues/lyd.ts

```diff
@@ -64,7 +65,8 @@ export function EvaluateLYD(
 					: fade
 					? Math.max(1000, fadeIn ? TimeFromFrames(fadeIn) : 0)
 					: CreateTimingEnable(parsedCue).enable.duration ?? undefined,
-				content: LydContent(config, file, lydType, fadeIn, fadeOut)
+				content: LydContent(config, file, lydType, fadeIn, fadeOut),
+				tags: [AdlibTags.ADLIB_FLOW_PRODUCER]
```

Comment:
Would be nice if we had a unitTest verifying this tag should be here, so we don't accidentally remove it down the line.
And if that test had a good name of why this tag is needed, we would then also remember why it is needed :)